bedrock/externalpages/templates/externalpages/pocket/base.html

```diff
@@ -0,0 +1,53 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    {% block shared_meta %}
+      <meta charset="utf-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
+      <meta name="description" content="{% block page_desc%}pocket_desc{% endblock %}"/>
+      <meta name="x-pocket-override-excerpt" content="{% block pocket_override %}{{ self.page_desc()}}{% endblock %}"/>
+      <meta itemprop="name" content="{{ self.page_title() }} | Pocket" />
+      <meta itemprop="description" content="{{ self.page_desc()}}"/>
+      <meta name="twitter:card" content="summary" />
+      <meta name="twitter:url" content="{% block page_twitter_url %}{% endblock %}" />
+      <meta name="twitter:title" content="{{ self.page_title() }} | Pocket" />
+      <meta name="twitter:description" content="{% block twitter_desc %}{% endblock %}"
+      />
+      <meta name="twitter:site" content="@pocket" />
+      <meta property="fb:app_id" content="131450656879143" />
+      <meta property="og:type" content="website" />
+      <meta property="og:url" content="{% block page_og_url %}{% endblock %}" />
+      <meta property="og:title" content="{{ self.page_title() }} | Pocket" />
+      <meta property="og:description" content="{{ self.page_desc()}}"/>
+      <meta property="og:site_name" content="Pocket" />
+      <meta name="next-head-count" content="6" />
+    {% endblock shared_meta%}
+
+
+      <!--[if !IE]><!-->
+    {# Page-specific styles, hidden from IE9 and lower #}
```

Comment:
I don't see a closing line for this conditional comment, so I'm not sure it's doing anything. The pocket pages also don't seem to have a default stylesheet for older browsers, so is this needed?